Resources/Textures/_RMC14/Structures/Furniture/rollerbeds.rsi/meta.json

```diff
@@ -38,16 +38,16 @@
       "name": "winched_stretcher",
       "delays": [
         [
-          0.1,
-          0.1,
-          0.1,
-          0.1,
-          0.1,
-          0.1,
-          0.1,
-          0.1,
-          0.1,
-          0.1,
+          0.3,
+          0.3,
+          0.3,
+          0.3,
+          0.3,
+          0.3,
+          0.3,
+          0.3,
+          0.3,
+          0.3,
```

Comment:
Is this intended? Does the animation play too quickly in comparison to 13 otherwise?

Reply:
Since I'm using the sprite component's auto-animate, I set the time of each frame so that it's total is similar to the medevac system's total time, which is 3 seconds